bluesky/plans.py

```diff
@@ -581,7 +581,7 @@ def tune_centroid(
     smaller step size until the minimum step size is reached.
     Rescans will be centered on the signal centroid
     (for $I(x)$, centroid$= \sum{I}/\sum{x*I}$)
-    with a scan range of 2*step_factor*step of current scan.
+    with original scan range reduced by `step_factor`.
```

Comment:
An annoying sphinx detail: single backticks render the contents in italics, not as code. You need ``step_factor`` or :data:`step_factor` to make sphinx happy. (GitHub Markdown behaves different than sphinx; it render single- or double-backticks the same way.)